docs/concept/dbsl.md

```diff
@@ -0,0 +1,354 @@
+# BanyanDB Query Language (BDSL)
```

Comment:
Please link this in the menu, as this is an important no matter in CLI, UI, and even in the future in API.

Reply:
The contents are good for me. Just one to confirm, as you are considering the API endpoint as the default context(default group, metrics, etc.).
Is the first implementation still CLI only?